src/settings/Api.ts

```diff
@@ -81,79 +94,72 @@ export class Api {
 		return axios.post(generateUrl('/apps/settings/') + '/settings/authorizedgroups/saveSettings', {
 			newGroups,
 			class: classname,
-		}).then((data) => data.data)
```

Comment:
Don't we need to keep it? `response.data` ?

Reply:
The return type is Promise<void> and the result is never used anywhere, so I removed it.

Reply:
Ah, then we should probably remove the return statement

Reply:
Or change the return type

Reply:
Returning the promise is fine otherwise it would be await. There is really no difference in that.